src/trainer.py

```diff
@@ -113,11 +118,19 @@ def fit(self, env: gym.Env, replay_buffer: ReplayBuffer, model: Dreamer) -> None
                     # logging
 
                 if current_step % self.model_save_interval == 0:
-                    self.save_checkpoint("/logs/...", model, world_optimizer, controller_optimizer)
+                    save_path = os.path.join(
+                        self.checkpoint_destination_path,
+                        f"{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}.ckpt",
+                    )
+                    self.save_checkpoint(save_path, model, world_optimizer, controller_optimizer)
 
                 current_step += 1
 
-        self.save_checkpoint("/logs/...", model, world_optimizer, controller_optimizer)
+        save_path = os.path.join(
+            self.checkpoint_destination_path,
+            f"{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}.ckpt",
```

Comment:
ファイル名はエピソード数やステップ数を記述することが適切です。その方が実験時のタイミングを一目で知ることができるためです。
```suggestion
            f"episode-{episode}_step-{current_step}.ckpt",
```